libs/langchain/langchain/retrievers/dashvector.py

```diff
@@ -0,0 +1,103 @@
+"""Wrapper around DashVector vector database."""
+
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+from uuid import uuid4
+
+from pydantic import root_validator
+
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForRetrieverRun,
+    CallbackManagerForRetrieverRun,
+)
+from langchain.docstore.document import Document
+from langchain.embeddings.base import Embeddings
+from langchain.schema import BaseRetriever
+
+
+class DashVectorRetriever(BaseRetriever):
```

Comment:
is there a different between this class and the functionality you'd get with DashVector.as_retriever()?

Reply:
No. Thanks for your reminder, I have removed it and add a `DashVectorHybridSearchRetriever` when this  feature is supported in the future.